### PR TITLE
Populate Organization ID for all the REST calls to the API Manager

### PR DESCRIPTION
**Pull request by VirajSalaka**
### Issues

Fixes #

### Automation tests
 - Unit tests added: No
 - Integration tests added: No

### Tested environments

Not Tested

---
#### Maintainers: Check before merge
- [ ] Assigned 'Type' label
- [ ] Assigned the project
- [ ] Validated respective github issues
- [ ] Assigned milestone to the github issue(s)

### CODE REVIEW

File: adapter/config/default_config.go [hunk 1]
```diff
@@ -326,6 +326,7 @@ var defaultConfig = &Config{
 		ServiceURLDeprecated: UnassignedAsDeprecated,
 		Username:             "admin",
 		Password:             "$env{cp_admin_pwd}",
+		OrganizationID:       "",
```
Comment (renuka-fernando) [inline adapter/config/default_config.go:329]: Let's add this in the config template file too. We can mention that this config is used for integration tests/local tests.

**Review by renuka-fernando — APPROVED** [review]

**Review by renuka-fernando — APPROVED** [review]

**Comment by choreo-cicd** [comment]
[succeeded]  : dev-deployment-v2 : 20240531.8

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : dev-deployment-v2 : 20240531.8

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : dev-deployment-v2 : 20240531.8

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : dev-deployment-v2 : 20240531.8

**Comment by choreo-cicd** [comment]
[failed] Controlplane cluster : stage-deployment-v2 : 20240603.3

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : stage-deployment-v2 : 20240603.3

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(CDP2-WU2) cluster : stage-deployment-v2 : 20240603.3

**Comment by choreo-cicd** [comment]
[failed] Dataplane(EastUS) cluster : stage-deployment-v2 : 20240603.3

**Comment by choreo-cicd** [comment]
[] Dataplane(NorthEU) cluster : stage-deployment-v2 : 20240603.3

**Comment by choreo-cicd** [comment]
[] Dataplane(CDP2-WU2) cluster : stage-deployment-v2 : 20240603.3

**Comment by choreo-cicd** [comment]
[] Dataplane(EastUS) cluster : stage-deployment-v2 : 20240603.3

**Comment by choreo-cicd** [comment]
[] Controlplane cluster : stage-deployment-v2 : 20240603.3

**Comment by choreo-cicd** [comment]
[succeeded] Controlplane cluster : prod-deployment-v2 : 20240604.1

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20240604.1

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(CDP2-WU2) cluster : prod-deployment-v2 : 20240604.1

**Comment by choreo-cicd** [comment]
[succeeded] Dataplane(EastUS) cluster : prod-deployment-v2 : 20240604.1

**Comment by choreo-cicd** [comment]
[] Dataplane(EastUS) cluster : prod-deployment-v2 : 20240604.2

**Comment by choreo-cicd** [comment]
[] Controlplane cluster : prod-deployment-v2 : 20240604.2

**Comment by choreo-cicd** [comment]
[] Dataplane(NorthEU) cluster : prod-deployment-v2 : 20240604.2

**Comment by choreo-cicd** [comment]
[] Dataplane(CDP2-WU2) cluster : prod-deployment-v2 : 20240604.2

**Pull request merged**